knime_extension/src/nodes/spatialstatistics.py

```diff
@@ -804,6 +804,12 @@ def execute(self, exec_context: knext.ExecutionContext, input_1, input_2):
         gdf.loc[:, "Local Getis-Ord G"] = lo.Gs
         gdf.loc[:, "p-value"] = lo.p_sim
         gdf.loc[:, "z-score"] = lo.z_sim
+        gdf.loc[:, "standardized Zs"] = lo.z_sim
```

Comment:
Why add another column if it contains the same value as the z-score column?

Reply:
Thanks, it's a bug. I just changed it and add more descriptions for the node.